packages/terra-functional-testing/lib/terra-dev-site/doc/terra-service/TerraService.doc.mdx

```diff
@@ -0,0 +1,64 @@
+# Terra Service
+
+The terra service integrates [axe accessibility](https://github.com/dequelabs/axe-core) testing into the test environment.
+
+## Installation
+
+The terra service should be installed as a development dependency.
```

Comment:
```suggestion
The Terra Service should be installed as a development dependency.
```

Reply:
I met halfway and uppercased all occurrences of `Terra`. I left `service` lowercase because of how it's being used in a sentence. It's uppercase when used as a title or single noun. `TerraService`. My experience with grammar is lacking so this could be wrong.

https://github.com/cerner/terra-toolkit/pull/416/commits/2e4d70c646089a29c41346d30db7336ab1a5f41a